github/event_types.go

```diff
@@ -1467,6 +1467,26 @@ type ReleaseEvent struct {
 	Org *Organization `json:"organization,omitempty"`
 }
 
+// RegistryPackageEvent represents activity related to GitHub Packages.
+// The Webhook event name is "registry_package".
+//
+// This event is triggered when a GitHub Package is published or updated.
+//
+// GitHub API docs: https://docs.github.com/en/webhooks/webhook-events-and-payloads#registry_package
+type RegistryPackageEvent struct {
```

Comment:
Could we please keep these structs sorted by name?

Reply:
sure on it

Reply:
fixed